aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/update_worker_helpers.py

```diff
@@ -131,18 +140,37 @@ def update_detector_version_for_detector_update(afd_client,
         common_helpers.update_tags(
             frauddetector_client=afd_client,
             afd_resource_arn=latest_dv_arn,
-            new_tags=model_helpers.get_tags_from_tag_models(model.Tags)
+            new_tags=model.Tags,
```

Comment:
Another bug, update tags gets tags from tag models, so no need to do so beforehand.